From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@dagood__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
Update app-insights-api-filtering-sampling.md

twooley committed:
new article for external monitoring

Commit:
devices43

twooley committed:
new article for external monitoring

MatkovIvan committed:
Add missing url

MatkovIvan committed:
Add syntax highlighting for code blocks

MatkovIvan committed:
Update deprecated code

MatkovIvan committed:
Replace ellipsis character

twooley committed:
new article for external monitoring

mahesh-unnikrishnan committed:
Update active-directory-ds-enable-using-powershell.md

twooley committed:
new article for external monitoring

twooley committed:
new article for external monitoring

MatkovIvan committed:
Update ms.date

anumjs committed:
Adding instructions for Java version

anumjs committed:
Fixed typo

anumjs committed:
Tweaked explanation

anumjs committed:
Changed author name

SnehaGunda committed:
UPdating register

twooley committed:
new article for external monitoring

twooley committed:
new article for external monitoring

kanshiG committed:
Update IP firewall for 0.0.0.0 rule

This was added recently in time for Connect and required by customers for allowing PaaS service to access Cosmos DB.

MatkovIvan committed:
Update pushes to Firebase Cloud Messaging

MatkovIvan committed:
Fix typo

MatkovIvan committed:
Fix formatting

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

kanshiG committed:
Fixing grammar and other changes.

barclayn committed:
adding a clarification warning to one of the steps

Commit:
Add example of setting the content type (MIME type)

nitinme committed:
adding more issues

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

dlepow committed:
HPC Pack AD: Portal updates

ktoliver committed:
edit pass: sql-database-troubleshoot-data-sync.md

ktoliver committed:
fix spacing in lists

ktoliver committed:
fix spacing in lists

barclayn committed:
testing is interactive works for powershell code snippets

SnehaGunda committed:
UPdating registration topic

SnehaGunda committed:
Updating registration

MikeDodaro committed:
Rebuild index.yml for landing page

anumjs committed:
Adding screenshot for Java

anumjs committed:
Adding screenshot for java version

anumjs committed:
Updated image for Java version

SnehaGunda committed:
More updates

Commit:
edit pass: active-directory-passwords-update-your-own-password

mmacy committed:
[aci] bump required CLI version

SnehaGunda committed:
More updates

spelluru committed:
Moving note up

rwike77 committed:
redirect

anumjs committed:
Fixed spacing

rwike77 committed:
removed article

rwike77 committed:
removed article

rwike77 committed:
updated links

nitinme committed:
fixing Acrolinx score

rwike77 committed:
removed article

rwike77 committed:
updated toc

rwike77 committed:
redirect

ktoliver committed:
Update sql-database-troubleshoot-data-sync.md

Add "Azure" to first instance of product name.

rwike77 committed:
updated links

rwike77 committed:
added text

iainfoulds committed:
Update ansible-create-vm.md

iainfoulds committed:
Update ansible-create-complete-vm.md

daveba committed:
saas-app-tutorial-list-update

twooley committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into twintmon2

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into deletefirstcluster

barbkess committed:
change authors

anumjs committed:
Changed formatting

twooley committed:
new article for external monitoring

v-thepet committed:
fix links

nitinme committed:
adding lines in between

Commit:
Add link to 'saas-tenancy-app-design-patterns.md', and add ## Next steps section of links.

nitinme committed:
one more line

Lisaco88 committed:
Merge pull request #28198 from rwike77/dotnet-tutorial

Dotnet tutorial

twooley committed:
new article for external monitoring

twooley committed:
new article for external monitoring

rwike77 committed:
fixed conflict

kgremban committed:
Update note

Commit:
Merge pull request #28197 from iainfoulds/patch-5

VM Ansible anchor links

rwike77 committed:
fix

PRMerger19 committed:
Merge pull request #28203 from barbkess/authors

change authors

twooley committed:
new article for external monitoring

PRMerger18 committed:
Merge pull request #28209 from kgremban/patch-1

Update note

PRMerger18 committed:
Merge pull request #26199 from MikeWasson/patch-1

Add ASP.NET Core example for ITelemetryInitializer

PRMerger14 committed:
Merge pull request #28188 from spelluru/adfauth117-2

Moving note up

Commit:
Incorp review comments, miscl edits

spelluru committed:
contributor or owner role

spelluru committed:
administrator also allowed

spelluru committed:
No Data Factory object

spelluru committed:
Cleanup

PRMerger16 committed:
Merge pull request #28214 from spelluru/adfqs1117

contributor or owner role

SnehaGunda committed:
updating cmdlet

AjayKumar-N committed:
(ForumsTeam) Incorrect forum link hyperlinked

In the referenced document: https://docs.microsoft.com/en-us/azure/storage/files/storage-files-faq
The "2.Azure Storage Forum" link points to incorrect forum for help. The hyperlink should point to Azure Storage forum : https://social.msdn.microsoft.com/forums/azure/en-US/home?forum=windowsazuredata

gsacavdm committed:
Remove # for code snippets

With the #, the code snippets' *Copy* functionality becomes useless since you can't paste into a terminal window.

gsacavdm committed:
Fixing links and next steps

Commit:
Merge pull request #28217 from gsacavdm/patch-3

Fixing links and next steps

Commit:
links to creating DSVM/HDI

Commit:
links to creating DSVM/HDI

Commit:
Merge pull request #28220 from hning86/master

add creating dsvm/hdi link to iris tutorial part 2

spelluru committed:
updated quickstart and tutorial

Commit:
update Git article

Commit:
update Git article

Commit:
update Git article

Commit:
update Git article

johnpapa committed:
updated the mongo connection string info

renamed conn string properties in the **environment** files. Added `databaseName`, too, It defaults to `admin`

spelluru committed:
one more pass on tutorial

Commit:
trfix04

gsacavdm committed:
Clarify preview subscription IDs

PRMerger20 committed:
Merge pull request #28225 from gsacavdm/patch-4

Clarify preview subscription IDs

PRMerger18 committed:
Merge pull request #28204 from v-thepet/newfix

production bug: fix links

spelluru committed:
fixes

PRMerger19 committed:
Merge pull request #28139 from MikeWasson/wasson/blob-set-mime-type-cli

Add example of setting the content type (MIME type)

PRMerger10 committed:
Merge pull request #27981 from mahesh-unnikrishnan/patch-15

Update active-directory-ds-enable-using-powershell.md

PRMerger20 committed:
Merge pull request #28226 from MarkusVi/trfix04

trfix04

GitHubber17 committed:
Merge pull request #28199 from MicrosoftDocs/saas-app-tutorial-list-udpate-complianceelf-turborater-schoox

saas-app-tutorial-list-update-complianceelf-turborater-schoox

GitHubber17 committed:
Merge pull request #28092 from anumjs/master

Added instructions for Java version

GitHubber17 committed:
Merge pull request #28205 from MightyPen/welcomebg-20171117-c3

Add link to 'saas-tenancy-app-design-patterns.md'.

GitHubber17 committed:
Merge pull request #28145 from nitinme/nitinme-databricks-updates

Adding a new TS issue

GitHubber17 committed:
Merge pull request #28190 from rwike77/deletefirstcluster

Delete first cluster article

GitHubber17 committed:
Merge pull request #27760 from MarkusVi/devices43

devices43

GitHubber17 committed:
Merge pull request #28167 from SnehaGunda/policy

updates to registration topic

GitHubber17 committed:
Merge pull request #28222 from hning86/master

update Git article

MatkovIvan committed:
Fix gradle syntax highlighting

MatkovIvan committed:
Update app-service-mobile-verify-android-sdk-version.md

MatkovIvan committed:
Update app-service-mobile-android-getting-started-with-push.md

rayne-wiselman committed:
updating redirects

dominicbetts committed:
Remove references to old portal

sanjeev3 committed:
Moved Video API from Concepts to QuickStarts

sanjeev3 committed:
Update toc.yml

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

PRMerger17 committed:
Merge pull request #28241 from dominicbetts/classic-portal

Remove references to old portal

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

spelluru committed:
formatting issue

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

barclayn committed:
final changes to app registration section

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

mimig1 committed:
Cosmos DB: Updated info about Table connection string

PRMerger20 committed:
Merge pull request #28126 from barclayn/keyvault-add-warning

adding a clarification warning to one of the steps

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into hpcad

dlepow committed:
json formatting fix

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into two-active-directory-password

Commit:
edit pass: active-directory-passwords-writeback

dlepow committed:
list fix

PRMerger12 committed:
Merge pull request #28246 from dksimpson/dean-380

Incorp review comments, miscl edits

rjagiewich committed:
Merge pull request #28221 from spelluru/adfustudy1118-2

updated quickstart and tutorial

dlepow committed:
list formatting

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rjagiewich committed:
Merge pull request #28239 from rayne-wiselman/rayne-november20

updating redirects

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

charwen committed:
Updated the providers and the locations

sanjeev3 committed:
Update video-moderation-api.md

charwen committed:
Updated the providers and the locations

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

mimig1 committed:
updated text

rjagiewich committed:
Merge pull request #28216 from gsacavdm/patch-2

Remove # for code snippets

v-alje committed:
Merge pull request #28248 from MicrosoftDocs/FromPublicRepo

From public repo

sanjeev3 committed:
Update video-moderation-api.md

sanjeev3 committed:
Update video-moderation-api.md

dlepow committed:
format fix -again

Lisaco88 committed:
Merge pull request #28169 from MikeDodaro/news-landing-page-2

Rebuild index.yml for landing page

Commit:
Merge pull request #28244 from mimig1/11202017tabletutorial

Cosmos DB: Updated info about Table connection string

Commit:
Merge pull request #28250 from charwen/patch-35

Updated the providers and the locations

Commit:
Merge pull request #28249 from charwen/patch-34

Updated the providers and the locations

v-shils committed:
Merge pull request #27887 from MatkovIvan/patch-3

Update deprecated code and add syntax highlighting for code blocks

dlepow committed:
one more time

dlepow committed:
format

spelluru committed:
Updates based on feedback

kgremban committed:
QUick fixes to c# tutorial

PRMerger13 committed:
Merge pull request #28253 from spelluru/adfustudy1120

Updates based on feedback

sanjeev3 committed:
Update video-moderation-api.md

ShawnJackson committed:
Merge pull request #28149 from ktoliver/sql-database-troubleshoot-data-sync

edit pass: sql-database-troubleshoot-data-sync.md

spelluru committed:
Updated the image

mimig1 committed:
Cosmos DB: Added important note about Table accts

CawaMS committed:
change sample code

ShawnJackson committed:
Update active-directory-passwords-update-your-own-password.md

GitHubber17 committed:
Merge pull request #28146 from dlepow/hpcad

HPC Pack AD: Portal updates

Commit:
Merge pull request #28255 from kgremban/nov20-csharpfix

QUick fixes to c# tutorial

johnpapa committed:
removed line 74

double whitespace lines.

Commit:
fixed broken link

PRMerger10 committed:
Merge pull request #28259 from CawaMS/release-connect-vsplat

change sample code

PRMerger17 committed:
Merge pull request #28223 from johnpapa/patch-16

updated the mongo connection string info

ShawnJackson committed:
Update active-directory-passwords-writeback.md

ShawnJackson committed:
Update active-directory-passwords-writeback.md

v-shils committed:
Merge pull request #27684 from twooley/twintmon2

new article for external monitoring

v-shils committed:
Merge pull request #28242 from sanjeev3/master

Updated Video Moderation API Article and TOC location

mimig1 committed:
Added another note to the create a database account section.

jennyhunter-msft committed:
Added Known Issues

- Added Known Issues section
- Added hotfix collection issue

jennyhunter-msft committed:
Typo fixes

- Updated minor typo fixes, including two fixes to "recursion"

Commit:
Merge pull request #28112 from kanshiG/patch-19

Update IP firewall for 0.0.0.0 rule

mimig1 committed:
Corrected warning

mimig1 committed:
Cosmos DB: Provided ETA on Cass approvals

ShawnJackson committed:
Merge pull request #28184 from LoriMerrick/two-active-directory-password

edit pass: two-active-directory-password

spelluru committed:
added troubleshooting steps

johnkemnetz committed:
Update monitoring-overview-activity-logs.md

spelluru committed:
fixes

Commit:
minor fix

GitHubber17 committed:
Merge pull request #28256 from spelluru/adfustudy1120-2

Updated the image

Commit:
date fix

Commit:
Merge pull request #28263 from jennyhunter-msft/patch-1

Added Known Issues

Commit:
Merge pull request #28268 from johnkemnetz/patch-1

Update monitoring-overview-activity-logs.md

spelluru committed:
minor

PRMerger20 committed:
Merge pull request #28269 from dkkapur/master

minor fix to monitoring overview doc

PRMerger16 committed:
Merge pull request #28215 from AjayKumar-N/patch-8

(ForumsTeam) Incorrect forum link hyperlinked

Lisaco88 committed:
Merge pull request #28117 from MatkovIvan/patch-4

Update pushes to Firebase Cloud Messaging

KumudD committed:
fixed email link

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into PrivateDNS1120

PRMerger17 committed:
Merge pull request #28258 from mimig1/11202017tablewarning

Cosmos DB: Added important note about Table accts

mmacy committed:
[aci] CLI version bump + output update

mimig1 committed:
Cosmos DB: Added note to create database section

PRMerger10 committed:
Merge pull request #28266 from mimig1/CassandraETA

Cosmos DB: Provided ETA on Cass approvals

PRMerger14 committed:
Merge pull request #28267 from spelluru/adfqs1120

added troubleshooting steps

ramankumarlive committed:
added s to Disk for SEO

added s to Disk for search engine optimization

spelluru committed:
delete resource group

ramankumarlive committed:
Update about-disks-and-vhds.md

PRMerger18 committed:
Merge pull request #28185 from mmacy/aci-cli-version-update

[aci] bump required CLI version

ramankumarlive committed:
Made changes for SEO

Made changes for search engine opitmization

PRMerger19 committed:
Merge pull request #28261 from hning86/master

fixed broken link

KumudD committed:
updated metadata

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into PrivateDNS1120

CarlRabeler committed:
xiaochen

PRMerger18 committed:
Merge pull request #28277 from spelluru/adfqs1120-3

delete resource group

PRMerger18 committed:
Merge pull request #28279 from ramankumarlive/patch-2

Made changes for SEO

Commit:
Merge pull request #28276 from ramankumarlive/patch-1

added s to Disk for SEO

PRMerger13 committed:
Merge pull request #28271 from mimig1/patch-167

Cosmos DB: Added note to create database section

PRMerger16 committed:
Merge pull request #28281 from CarlRabeler/restore

xiaochen

ktoliver committed:
Merge pull request #28275 from KumudD/PrivateDNS1120

Fix email link
